src/applications/disability-benefits/2346/app-entry.jsx

```diff
@@ -1,12 +1,10 @@
 import 'platform/polyfills';
 import startApp from 'platform/startup';
 import manifest from './manifest.json';
-import reducer from './reducers';
```

Comment:
I thought we still needed to import and include the form system's reducer in our `startApp`. Is that not the case?

Reply:
We do!  I already added it back in by the time you left this comment :-P Mistake on my part